packages/react-components/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -10,17 +10,33 @@ import {
   PRIMARY_STORY,
   Stories,
 } from '@storybook/addon-docs';
+import { makeStyles } from '@fluentui/react-components';
```

Comment:
```suggestion
import { makeStyles } from ../../index';
```

Reply:
```suggestion
import { makeStyles } from '@fluentui/react-make-styles';
```

Reply:
importing directly from `react-components` doesn't work because of the TS issues in stories that we've encountered.

In this case instead of ts-ignore the relative import is perfectly fine, or importing from the specific package